release-npm/action.yml

```diff
@@ -0,0 +1,198 @@
+        echo "GROUP_ID=$(jq '.jahia.maven.groupId' package.json | cut -d\" -f2)" >> $GITHUB_ENV
+
+    - name: Getting latest changes from the remote branch
+      shell: bash
+      run: |
+        git fetch --all
+        git pull origin ${{ inputs.primary_release_branch }} --quiet
+
+    - name: Delete github tag ${{ inputs.release_version }}
+      shell: bash
+      run: |
+        git config user.email ${{ inputs.git_user_email }}
+        git config user.name ${{ inputs.git_user_name }}
+        git tag --delete ${{ inputs.release_version }}
+        git push origin :refs/tags/${{ inputs.release_version }}
+
+    - name: Remove any previous changes, this is required by mvn release prepare
+      shell: bash
+      run: |
+        git reset --hard
+        git checkout ${{ inputs.primary_release_branch }}
+
+    - name: Added some debug details
+      shell: bash
+      run: |
+        git branch
+        echo "---"
+        git status
+        echo "---"
+        git show --summary
+
+    - name: Release prepare version for ${{ inputs.release_version }}
+      shell: bash
+      run: |
+        TAG_VERSION=$(echo ${{ inputs.release_version }})
+        FINAL_RELEASE_VERSION=$(echo ${{ inputs.release_version }} | tr '_' '.' | cut -d'-' -f 1)
+        echo "FINAL_RELEASE_VERSION=$FINAL_RELEASE_VERSION" >> $GITHUB_ENV
+        MAJOR_VERSION=`echo $FINAL_RELEASE_VERSION | cut -d. -f1`
+        MINOR_VERSION=`echo $FINAL_RELEASE_VERSION | cut -d. -f2`
+        REVISION=`echo $FINAL_RELEASE_VERSION | cut -d. -f3`
+        if [[ "${{ inputs.primary_release_branch }}" == "master"  || "${{ inputs.primary_release_branch }}" == "main"  ||  "${{ inputs.primary_release_branch }}" =~ ^[0-9]_x$ ]]; then
+          NEXT_MINOR_VERSION=`expr $MINOR_VERSION + 1`
+          NEXT_DEVELOPMENT_VERSION="$MAJOR_VERSION.$NEXT_MINOR_VERSION.0"
+        else
+          NEXT_REVISION=`expr $REVISION + 1`
+          NEXT_DEVELOPMENT_VERSION="$MAJOR_VERSION.$MINOR_VERSION.$NEXT_REVISION"
+        fi
+        echo "NEXT_DEVELOPMENT_VERSION=$NEXT_DEVELOPMENT_VERSION" >> $GITHUB_ENV
+
+    - name: Update package.json
+      shell: bash
+      run: |
+        jq '.version |= "${{ env.FINAL_RELEASE_VERSION }}"' package.json > txt.json && jq '.' txt.json > package.json && rm txt.json
+
+        if [ ! ${{ env.FINAL_RELEASE_VERSION }} -eq ${{ env.MODULE_VERSION }} ]
+        then
+          git config user.name ${{ inputs.github_user_name }}
+          git config user.email ${{ inputs.github_user_email }}
+          git remote set-url origin https://x-access-token:${{ inputs.github_api_token }}@github.com/${{ inputs.github_slug}}
+          git add package.json
+          git commit -m "[skip ci] prepare release $FINAL_RELEASE_VERSION"
+          git push
+        fi
+        echo "MODULE_VERSION=$(jq '.version' package.json | cut -d\" -f2)" >> $GITHUB_ENV
+
+    - name: Prepare the module
+      shell: bash
+      run: yarn
```

Comment:
Same comment than above about supporting a module_path parameter across the entire action